packages/font/src/google/get-google-fonts-url.ts

```diff
@@ -50,14 +50,36 @@ export function getGoogleFontsUrl(
     '+'
   )}`
 
+  function sortVariantValues(valA: string, valB: string) {
```

Comment:
Can we export this util looks like it doesn't rely on other dependencies, and add a unit test for this? 

Could potentialy rename to `sortFontsVariantValues` as it's only related to font

Reply:
Certainly, I'll rename it to `sortFontsVariantValues` that would be a more relevant name, also write unit test for this function and handle prettier formatting issue.

Yes, it doesn't rely on other dependencies and if we want to export this, we can.

I have one question.

In case we want to export this, where should I put this function, should I create a new file named `sort-fonts-variant-values.ts` and write a unit test in a file named `sort-font-variant-values.test.ts` in the same directory or do you want to suggest something else?

Reply:
`sort-fonts-variant-values.ts` sounds good